README.md

```diff
@@ -1,2 +1,17 @@
-# thermal_base
-Decoding and common processing for thermographs / thermograms. Currently supports FLIR's TIFF and PNG encoded metadata, and DJI-encoded metadata
+# Base codes for Thermography
+
+A python package for decoding and common processing for thermographs / thermograms. Currently supports FLIR's TIFF and PNG encoded metadata, and DJI-encoded metadata
+
+## Install
+Install this package with `pip install git+ssh://git@github.com/detecttechnologies/thermal_base.git@main`
```

Comment:
Can you also add a `requirements.txt` file and mention it in the Readme's installation instructions?

I believe apart from the `requirements.txt` file, they'll also need exiftool for this package, you can mention that the user can `sudo apt-get install exiftool` or install it from <insert link to windows download location>

Reply:
The requirements are listed within setup.py. pip installing this package will install the dependancies also. Do you still want me to add a `requirements.txt` file?

Reply:
Yes please, if someone is building it by cloning the repo and not via pip, then it'll be required. Also, the `requirements.txt` file also usually mentions the exact versions of the different libraries against which the overall project has been tested (ex: opencv==4.2.1)